tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -421,6 +421,24 @@ Uniforms can also be assigned default values:
     uniform vec4 some_vector = vec4(0.0);
     uniform vec4 some_color : hint_color = vec4(1.0);
 
+Global constants
+----------------
+
+Like the uniforms global constants are shared between all shader stages. However they are invisible outside of shader and must be initialized at declaration.
```

Comment:
Instead say: "Like uniforms, global constants..."
and
"However, they are not accessible outside of the shader..."